package.json

```diff
@@ -79,6 +80,7 @@
     "graphql": "15.1.0",
     "graphql-middleware": "4.0.2",
     "graphql-scalars": "1.1.5",
+    "graphql-subscriptions": "^1.1.0",
```

Comment:
suggestion: This should be in `dependencies` since it will be used during runtime and not just during build time